etc/config.xml

```diff
@@ -149,6 +149,7 @@
                 <allowspecific>0</allowspecific>
                 <sort_order>7</sort_order>
                 <session_validity>3</session_validity>
+                <ship_before>5</ship_before>
```

Comment:
Why is the default 5? Do we need to mention it in the admin form?

Reply:
@acampos1916 It was also defaulting to 5 days before the field as the previous HPP config was removed. However, this field is used to calculate the expiry date for the offline payment methods such as 7-Eleven or Boleto. Other than that, it is not a mandatory field.